scipy/interpolate/src/__fitpack.cc

```diff
@@ -1,4 +1,6 @@
 #include <string>
+#include <vector>
```

Comment:
```suggestion
#include <cstdint>
#include <vector>
```
fir use of, e.g., `int64_t`.